blockdata/constants.lisp

```diff
@@ -1,13 +1,16 @@
 (module blockdata)
 
-(defconst 
-  CT_MAX_FOR_BLOCKDATA    6 ;;need to update the lookup into wcp if changed
-  ROW_SHIFT_COINBASE      0
-  ROW_SHIFT_TIMESTAMP     1
-  ROW_SHIFT_NUMBER        2
-  ROW_SHIFT_DIFFICULTY    3
-  ROW_SHIFT_GASLIMIT      4
-  ROW_SHIFT_CHAINID       5
-  ROW_SHIFT_BASEFEE       6)
-
+(defconst
+  ;;
+  ;; CT MAX FOR BLOCKDATA COLUMNS
+  ;;
+  CT_MAX_CB       1
+  CT_MAX_TS       2
+  CT_MAX_NB       2
+  CT_MAX_DF       1
+  CT_MAX_GL       5
+  CT_MAX_ID       1
+  CT_MAX_BF       1
+  CT_MAX_DEPTH    13
```

Comment:
I'm not sure we can do this, _but if we can_ we should define `CT_MAX_DEPTH` as the sum of the other `CT_MAX`'s.